tests/test_api.py

```diff
@@ -648,7 +657,7 @@ def test_length_and_hash_validation(self):
 
             # Test wrong algorithm format (sslib.FormatError)
             snapshot_metafile.hashes = {
-                256: "8f88e2ba48b412c3843e9bb26e1b6f8fc9e98aceb0fbaa97ba37b4c98717d7ab"
+                256: "8f88e2ba48b412c3843e9bb26e1b6f8fc9e98aceb0fbaa97ba37b4c98717d7ab"  # type: ignore[dict-item]
```

Comment:
Here we receive the following mypy warning:
```
tests/test_api.py:660: error: Dict entry 0 has incompatible type "int": "str";
expected "str": "str"  [dict-item]
```
This error hints we have annotated the `snapshot_metafile.hashes` as a `Dict[str, str]`, but we
are using it as `Dict[int, str]`.
This is expected behavior here as we purposely test an invalid hash algorithm.